### PR TITLE
docs: storybook examples

### PR DESCRIPTION
**Pull request by jamime**
DO NOT MERGE.

### CODE REVIEW

File: src/components/example/example.component.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  onClick: PropTypes.func.isRequired,
+  /**
+   * Apply the super power button styling
+   */
+  enableSuperPower: PropTypes.bool
+};
+
+Example.defaultProps = {
+  enableSuperPower: false
+};
```
Comment (dawidzarzycki) [inline src/components/example/example.component.js:21]: What about moving defaultProps to the top?
```js
export const Example = ({ children, onClick, enableSuperPower = false }) => {
```

File: src/components/example/example.test.stories.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      disable: true
+    }
+  }
+};
+
+export const chromatic = () => (
+  <> <p>Story can have any name</p>
+    <Example>An example story for chromatic automation</Example>
+  </>
+);
```
Comment (dawidzarzycki) [inline src/components/example/example.test.stories.js:21]:
```suggestion
  <> 
    <p>Story can have any name</p>
    <Example>An example story for chromatic automation</Example>
  </>
);
```

File: src/components/example/example.test.stories.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export const cypress = () => (
+  <> <p>Story can have any name</p>
+    <Example>An example story for cypress automation</Example>
+  </>
+);
```
Comment (dawidzarzycki) [inline src/components/example/example.test.stories.js:27]:
```suggestion
export const cypress = () => (
  <> 
    <p>Story can have any name</p>
    <Example>An example story for cypress automation</Example>
  </>
);
```

File: src/components/example/example.component.js (hunk 1 above)
Comment (jamime) [reply src/components/example/example.component.js:21]: yep, we should do this.

**Pull request closed without merging**